app/src/main/res/values/strings.xml

```diff
@@ -377,9 +377,9 @@
     <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
     <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="autofill_auto_search_title">Auto search</string>
-    <string name="autofill_auto_search_summary">Automatically suggest search results from the web domain or application Id</string>
+    <string name="autofill_auto_search_summary">Automatically suggest search results from the web domain or app ID</string>
```

Comment:
applicationId is a standardized variable, you wanted to split it in two, OK. But it no longer corresponds to the variable if we remove letters. https://developer.android.com/studio/build/application-id

Reply:
Strange, I prefer "applicationId" over what looks like it is a spelling error "application Id". That is also something not used in the link.
The link uses "Application ID" to describe "applicationID". What does this do btw?

Reply:
In this case "application ID", "Id" is a code convention, second letter is a small case.